ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java

```diff
@@ -296,5 +275,41 @@ private void updateTestWorld() {
 
     }
 
+    /**
+     * Update the steering direction.
+     * @param orientation 
+     */
+    public void steer(float orientation){
+        steering = orientation;
+    }
+
+
+    /**
+     * Handles everything that happens when the Envrironment state is detached from the main application.
+     */
+    @Override
+    public void cleanup() {
+        super.cleanup();
+    }
```

Comment:
Why do you need to override the super cleanup method if that's the only thing you're calling?